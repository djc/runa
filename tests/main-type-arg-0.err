main-type-arg-0.rns [1.16]: 1st argument to main() must be of type &Str
def main(name: Str, args: Array[Str]):
               ^
