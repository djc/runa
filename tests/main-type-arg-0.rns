def main(name: Str, args: array[Str]):
	print(x)
