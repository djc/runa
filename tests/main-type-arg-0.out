tests/main-type-arg-0.rns [1.16]: 1st argument to main() must be of type &str
def main(name: str, args: array[str]):
               ^
